plugins/node/opentelemetry-instrumentation-aws-sdk/test/load-instrumentation.ts

```diff
@@ -21,12 +21,25 @@
  * specific test. We instead instantiate a single instrumentation instance here to
  * use within all tests.
  */
+import { registerInstrumentationTesting } from '@opentelemetry/contrib-test-utils';
 import {
-  initMeterProvider,
-  registerInstrumentationTesting,
-} from '@opentelemetry/contrib-test-utils';
+  AggregationTemporality,
+  InMemoryMetricExporter,
+  MeterProvider,
+  PeriodicExportingMetricReader,
+} from '@opentelemetry/sdk-metrics';
 import { AwsInstrumentation } from '../src';
 
 export const instrumentation = new AwsInstrumentation();
-export const metricReader = initMeterProvider(instrumentation);
```

Comment:
I realized this pattern seemed to not work for multiple tests. I did try passing `DELTA` temporality in the `MetricReader` constructor since I thought it would fix it but it didn't. So I changed to a pattern inspired by what's used in Java

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/testing-common/src/main/java/io/opentelemetry/instrumentation/testing/LibraryTestRunner.java#L122